app/assets/images/stembolt_logo.svg

```diff
@@ -0,0 +1,33 @@
+                        <path d="M140.3115,9.015 C140.3115,12.177 138.015,14.56 134.969,14.56 C131.944,14.56 129.574,12.102 129.574,8.969 L129.574,8.917 C129.574,5.7595 131.872,3.372 134.9165,3.372 C137.9405,3.372 140.3115,5.83 140.3115,8.969 L140.3115,9.0155 L140.3115,9.015 Z M144.0365,8.917 C144.0365,3.905 140.138,0.1245 134.969,0.1245 C129.856,0.1245 125.848,4.007 125.848,8.9685 L125.848,9.015 C125.848,14.0315 129.747,17.8115 134.9165,17.8115 C140.0315,17.8115 144.0365,13.9265 144.0365,8.969 L144.0365,8.917 L144.0365,8.917 Z"></path>
+                        <path d="M166.391,14.843 C166.4695,14.694 166.4775,14.559 166.419,14.462 C166.359,14.367 166.234,14.311 166.064,14.311 L159.036,14.311 C158.512,14.311 158.086,13.886 158.086,13.3635 L158.086,1.1015 C158.086,0.7265 157.78,0.42 157.405,0.42 L153.762,0.42 C153.663,0.42 153.5885,0.455 153.542,0.525 C153.467,0.632 153.467,0.81 153.538,0.999 L154.24,2.869 C154.405,3.301 154.538,4.035 154.538,4.494 L154.538,16.835 C154.538,17.21 154.8405,17.5125 155.2155,17.5125 L164.3255,17.5125 C164.718,17.5125 165.2055,17.214 165.383,16.8595 L166.3915,14.8425 L166.391,14.843 Z"></path>
+                        <path d="M186.091,0.9985 C186.1495,0.8415 186.141,0.696 186.071,0.5855 C185.996,0.479 185.859,0.4205 185.69,0.4205 L172.932,0.4205 C172.762,0.4205 172.627,0.479 172.553,0.5855 C172.478,0.6925 172.4745,0.838 172.533,0.999 L173.293,2.999 C173.432,3.362 173.8795,3.6725 174.2745,3.6725 L176.5845,3.6725 C177.1095,3.6725 177.538,4.0975 177.538,4.626 L177.538,16.836 C177.538,17.211 177.8405,17.5135 178.2155,17.5135 L180.4055,17.5135 C180.778,17.5135 181.0865,17.211 181.0865,16.836 L181.0865,4.625 C181.0865,4.0975 181.5115,3.672 182.034,3.672 L184.366,3.672 C184.756,3.672 185.201,3.362 185.338,2.9985 L186.09,0.9985 L186.091,0.9985 Z"></path>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
We are missing a new line char at the end of this file. I would find a way to configure your .vimrc file to do this on save or something.